app/src/main/java/com/epfl/beatlink/model/library/UserPlaylist.kt

```diff
@@ -8,6 +8,6 @@ data class UserPlaylist(
     val playlistCover: String,
     val playlistName: String, // mandatory
     val playlistPublic: Boolean = false,
-    val playlistSongs: List<SpotifyTrack>, // list of ids of SpotifyTrack
+    val playlistTracks: List<SpotifyTrack>, // list of ids of SpotifyTrack
```

Comment:
The comment on playlistTracks specifies "list of ids of SpotifyTrack" but based on the type List<SpotifyTrack>, it appears to be a list of SpotifyTrack objects rather than their IDs. I suggest either removing the comment or updating it to correctly describe the content.